pkg/cmd/pr/edit/edit.go

```diff
@@ -104,6 +108,7 @@ func NewCmdEdit(f *cmdutil.Factory, runF func(*EditOptions) error) *cobra.Comman
 
 	cmd.Flags().StringVarP(&opts.Editable.Title.Value, "title", "t", "", "Set the new title.")
 	cmd.Flags().StringVarP(&opts.Editable.Body.Value, "body", "b", "", "Set the new body.")
+	cmd.Flags().StringVarP(&opts.Editable.Base.Value, "base", "a", "", "The name of the branch you want your changes pulled into.")
```

Comment:
1. The shorthand should be `B`, since we also use that shorthand in `pr create`.
2. If you additionally enclose the word `` `branch` `` in backticks, the flag help rendering will be nicer.
3. The description for the flag should reveal more about what will happen if you use it; e.g. “Change the base branch for this pull request”